### DISCUSSION TITLE
Label next to requested notebook status

### DISCUSSION THREAD

**Discussion by Dennis-MSI** [snipe/snipe-it#11638] [Ideas]
It would be nice if similar to Ready to deploy, the Requested status got a little label when an asset has been requested. 
For Ready to deploy, it shows the "Deployed" tag.

![image](https://user-images.githubusercontent.com/96120758/183683133-c81fa7ea-9d28-42fb-9b08-4b6a8c40a987.png)

**Comment by Delirium-Sama** [comment]
Yeah nice shot i'm hesiting of using the request system for now because it feel buggy
